composer.json

```diff
@@ -36,7 +36,8 @@
         "misd/linkify": "~1.1.2",
         "league/flysystem-memory": "^1.0",
         "htmlawed/htmlawed": "^1.1.22",
-        "imagine/imagine": "^0.6.3"
+        "imagine/imagine": "^0.6.3",
+        "bower-asset/jquery-treeview": "^1.4"
```

Comment:
no like... i know why you did this, but still i do not like it... can accept for now, but in an ideal world this should stay with the plugin composer..